### ISSUE TITLE
Get-AzureRmApplicationGateway : Missing type map configuration or unsupported mapping.

### ISSUE THREAD

**Issue by inghak** [Azure/azure-powershell#3266]
### Cmdlet(s)
Get-AzureRmApplicationGateway
### PowerShell Version
PSVersion                      5.1.14393.206
### Module Version
Script     3.2.0      AzureRM.Network 
### OS Version
OS: 10.0.14393.206 Windows 
### Description
Executing: Get-AzureRmApplicationGateway
Error occurs with or without initial parameters. I.e:

Get-AzureRmApplicationGateway
Get-AzureRmApplicationGateway -Name NameOfAag -ResourceGroupName NameOfResourceGroup

Get the following error: 

Get-AzureRmApplicationGateway : Missing type map configuration or unsupported mapping.
Mapping types:
Int32 -> UInt32
System.Int32 -> System.UInt32
Destination path:
PSApplicationGateway.Probes.Probes.Probes0[0].Interval.Interval
Source value:
15
At line:1 char:1
+ Get-AzureRmApplicationGateway
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzureRmApplicationGateway], AutoMapperMappingException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Network.GetAzureApplicationGatewayCommand

### Debug Output
DEBUG: 10.31.05 - GetAzureApplicationGatewayCommand begin processing with ParameterSet '__AllParameterSets'.
DEBUG: 10.31.05 - using account id 'haking@spv.no'...
DEBUG: [Common.Authentication]: Authenticating using Account: 'haking@spv.no', environment: 'AzureCloud', tenant: '417918a5-f594-44e0-ba27-23112940bde5'
DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05:  - TokenCache: Deserialized 1 items to token cache.

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05: 04b79127-bfdb-4f75-9f28-2f2dbac37cd3 - AcquireTokenHandlerBase: === Token Acquisition started:
	Authority: https://login.microsoftonline.com/417918a5-f594-44e0-ba27-23112940bde5/
	Resource: https://management.core.windows.net/
	ClientId: 1950a258-227b-4e31-a9cf-717495945fc2
	CacheType: Microsoft.IdentityModel.Clients.ActiveDirectory.TokenCache (1 items)
	Authentication Target: User
	

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Verbose: 1 : 
DEBUG: 12/08/2016 09:31:05: 04b79127-bfdb-4f75-9f28-2f2dbac37cd3 - TokenCache: Looking up cache for a token...

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05: 04b79127-bfdb-4f75-9f28-2f2dbac37cd3 - TokenCache: An item matching the requested resource was found in the cache

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Verbose: 1 : 
DEBUG: 12/08/2016 09:31:05: 04b79127-bfdb-4f75-9f28-2f2dbac37cd3 - TokenCache: 23.3871023983333 minutes left until token in cache expires

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05: 04b79127-bfdb-4f75-9f28-2f2dbac37cd3 - TokenCache: A matching item (access token or refresh token or both) was found in the cache

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05: 04b79127-bfdb-4f75-9f28-2f2dbac37cd3 - AcquireTokenHandlerBase: === Token Acquisition finished successfully. An access token was retuned:
	Access Token Hash: kANkFrL6znJ1OAivNElejpVazILoCm8hIF5gXVhQbk8=
	Refresh Token Hash: VkMjHjn/FmZapLpkxN42n8YcawNuulN+F4yFqaJdVDg=
	Expiration Time: 12/08/2016 09:54:29 +00:00
	User Hash: 3HViLS4RStBOmWQRtOShUvDCFx/WYYMlmnfT1SvHK8c=
	

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05:  - TokenCache: Serializing token cache with 1 items.

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05: 990cd7df-b0b6-4ccd-a9d4-adb3d06a4d72 - AcquireTokenHandlerBase: === Token Acquisition started:
	Authority: https://login.microsoftonline.com/417918a5-f594-44e0-ba27-23112940bde5/
	Resource: https://management.core.windows.net/
	ClientId: 1950a258-227b-4e31-a9cf-717495945fc2
	CacheType: Microsoft.IdentityModel.Clients.ActiveDirectory.TokenCache (1 items)
	Authentication Target: User
	

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Verbose: 1 : 
DEBUG: 12/08/2016 09:31:05: 990cd7df-b0b6-4ccd-a9d4-adb3d06a4d72 - TokenCache: Looking up cache for a token...

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05: 990cd7df-b0b6-4ccd-a9d4-adb3d06a4d72 - TokenCache: An item matching the requested resource was found in the cache

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Verbose: 1 : 
DEBUG: 12/08/2016 09:31:05: 990cd7df-b0b6-4ccd-a9d4-adb3d06a4d72 - TokenCache: 23.3869185983333 minutes left until token in cache expires

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05: 990cd7df-b0b6-4ccd-a9d4-adb3d06a4d72 - TokenCache: A matching item (access token or refresh token or both) was found in the cache

DEBUG: Microsoft.IdentityModel.Clients.ActiveDirectory Information: 2 : 
DEBUG: 12/08/2016 09:31:05: 990cd7df-b0b6-4ccd-a9d4-adb3d06a4d72 - AcquireTokenHandlerBase: === Token Acquisition finished successfully. An access token was retuned:
	Access Token Hash: kANkFrL6znJ1OAivNElejpVazILoCm8hIF5gXVhQbk8=
	Refresh Token Hash: VkMjHjn/FmZapLpkxN42n8YcawNuulN+F4yFqaJdVDg=
	Expiration Time: 12/08/2016 09:54:29 +00:00
	User Hash: 3HViLS4RStBOmWQRtOShUvDCFx/WYYMlmnfT1SvHK8c=
	

DEBUG: ============================ HTTP REQUEST ============================

HTTP Method:
GET

Absolute Uri:
https://management.azure.com/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag?api-version=2016-09-0
1

Headers:
x-ms-client-request-id        : 41b244da-c7bc-4b0f-8935-9ac9b4d789fe
accept-language               : en-US

Body:

DEBUG: ============================ HTTP RESPONSE ============================

Status Code:
OK

Headers:
Pragma                        : no-cache
x-ms-request-id               : bbcbdba9-a5ae-4d6a-9891-78004624213b
Strict-Transport-Security     : max-age=31536000; includeSubDomains
Cache-Control                 : no-cache
ETag                          : W/"32093f62-12bc-467e-8604-b9eb0fd0ef47"
Server                        : Microsoft-HTTPAPI/2.0,Microsoft-HTTPAPI/2.0
x-ms-ratelimit-remaining-subscription-reads: 13625
x-ms-correlation-request-id   : 27a3fbbd-da82-4e5f-9174-680726a9ab33
x-ms-routing-request-id       : WESTEUROPE:20161208T093106Z:27a3fbbd-da82-4e5f-9174-680726a9ab33
Date                          : Thu, 08 Dec 2016 09:31:05 GMT

Body:
{
  "name": "s-secsvc-aag",
  "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag",
  "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
  "type": "Microsoft.Network/applicationGateways",
  "location": "westeurope",
  "properties": {
    "provisioningState": "Succeeded",
    "resourceGuid": "a0dac816-c140-4bf7-9d09-bde18af9f6cf",
    "sku": {
      "name": "Standard_Small",
      "tier": "Standard",
      "capacity": 1
    },
    "gatewayIPConfigurations": [
      {
        "name": "s-secsvcAgIPConfig",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/gatewayIPConfigurations/s-secsvcAg
IPConfig",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "subnet": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/SPVNETWORK/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/PRISM_Frontend_AG"
          }
        }
      }
    ],
    "sslCertificates": [
      {
        "name": "s-secsvcAgSslCert",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/sslCertificates/s-secsvcAgSslCert"
,
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "publicCertData": "MIIFWzCCBA+gAwIBAgITMQAAABAWqRxfyewxSAAAAAAAEDBBBgkqhkiG9w0BAQowNKAPMA0GCWCGSAFlAwQCAQUAoRwwGgYJKoZIhvcNAQEIMA0GCWCGSAFlAwQCAQUAogMCASAwTzESMBAGCgmSJomT8ixkARkWAm5
vMRMwEQYKCZImiZPyLGQBGRYDc3B2MSQwIgYDVQQDExtTcGFyZWJhbmtlbiBWZXN0IElzc3VpbmcgQ0EwHhcNMTYwOTEzMTMxMDI5WhcNMTgwOTEzMTMxMDI5WjApMScwJQYDVQQDEx5zZWNzdmMuc3lzdGVtdGVzdC5zcHYuc2VydmljZXMwggEiMA0GCSq
GSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDFbc3xv27Pfdc9GkTLIP+9SKKrdfN1fs/ckX46VlznjqkcbPR07gf6es+Khni4bs4o9wHL3xG8AHnEwY/OrJwLf7dWkeFDfLrTQVq1mVovDriZaprFdRgW9EUdp9PFXZiCBbKW7cSH95oTmP+G3yIQ24Y4gzpLPX4
9wB0RHNUQjCDkM6iLb6NNvi1IEZeNDLQ17HcoWlt4H7yhlzK1cp4iqrnu86l1vR2xDtaX4zpqUeZCDlCcaEnGB0WSw4MxiI5991uK1GuW3f8k+FUrJ0X3q+ngX4WWyrxyh6xodN4/V9AXJkuBIN2h6Yf1VfjUEhh/wzTHgQVr+b2xdGxQ2UwTAgMBAAGjggH
sMIIB6DA+BgkrBgEEAYI3FQcEMTAvBicrBgEEAYI3FQiE37V2g+K6PoehgwuC688whZTSc4Eth7DrYoett38CAWQCAR8wEwYDVR0lBAwwCgYIKwYBBQUHAwEwCwYDVR0PBAQDAgWgMBsGCSsGAQQBgjcVCgQOMAwwCgYIKwYBBQUHAwEwHQYDVR0OBBYEFJY
JK6kjI96qw6B3xooBnG8nB0YiMB8GA1UdIwQYMBaAFDvjx6742aEc9XSLXmlC8u3E2rjHMFYGA1UdHwRPME0wS6BJoEeGRWh0dHA6Ly9wa2kuc3B2ZXN0Lm5vL0NlcnRFbnJvbGwvU3BhcmViYW5rZW4lMjBWZXN0JTIwSXNzdWluZyUyMENBLmNybDCBzgY
IKwYBBQUHAQEEgcEwgb4wgbsGCCsGAQUFBzAChoGubGRhcDovLy9DTj1TcGFyZWJhbmtlbiUyMFZlc3QlMjBJc3N1aW5nJTIwQ0EsQ049QUlBLENOPVB1YmxpYyUyMEtleSUyMFNlcnZpY2VzLENOPVNlcnZpY2VzLENOPUNvbmZpZ3VyYXRpb24sREM9c3B
2LERDPW5vP2NBQ2VydGlmaWNhdGU/YmFzZT9vYmplY3RDbGFzcz1jZXJ0aWZpY2F0aW9uQXV0aG9yaXR5MEEGCSqGSIb3DQEBCjA0oA8wDQYJYIZIAWUDBAIBBQChHDAaBgkqhkiG9w0BAQgwDQYJYIZIAWUDBAIBBQCiAwIBIAOCAQEAZuX8Fmylid6RlOY
MEVC0kTlNs34GZZHVrTGVSjIqdW0SB82blUPE1potHD+mnmTtc7NVt7UuuWBlielja93d3gWnO53dP9MeqRJcN6KfBcpSNfejjc/W+Sgm2V3HJIll3y28X4kOCFyupB5fMkOmd+tzGbB5YVwAiOIwQaFeSpem5tgDwRDIXJ6nfNCoV7XXv1bJEtS4RJx9kCE
/O5vI0PSdBlxH6VlCb27t/aXuDMRWFSYNk1gIwjy0uzkCaNzjz3WJukBCxfKiw0BaYKCFBotttTkLU0WatpPmC/J+V6iwFR8IlM9UFukC23FwG8bmJtVQ9pUhbyWWQJVgZCspyA==",
          "httpListeners": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/httpListeners/s-secsvcAgHttp
Listener"
            }
          ]
        }
      }
    ],
    "authenticationCertificates": [
      {
        "name": "s-secsvcAgAuthenticationCert",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/authenticationCertificates/s-secsv
cAgAuthenticationCert",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "data": "(I have anonymized this, but here is cert listed)",
          "backendHttpSettings": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/backendHttpSettingsCollectio
n/s-secsvcAgBackendHttpSettings"
            }
          ]
        }
      }
    ],
    "frontendIPConfigurations": [
      {
        "name": "s-secsvcAgFrontendIP",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/frontendIPConfigurations/s-secsvcA
gFrontendIP",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "privateIPAddress": "10.206.22.6",
          "privateIPAllocationMethod": "Static",
          "subnet": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/SPVNETWORK/providers/Microsoft.Network/virtualNetworks/VNet1/subnets/PRISM_Frontend_AG"
          },
          "httpListeners": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/httpListeners/s-secsvcAgHttp
Listener"
            }
          ]
        }
      }
    ],
    "frontendPorts": [
      {
        "name": "s-secsvcAgFrontendPort",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/frontendPorts/s-secsvcAgFrontendPo
rt",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "port": 443,
          "httpListeners": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/httpListeners/s-secsvcAgHttp
Listener"
            }
          ]
        }
      }
    ],
    "backendAddressPools": [
      {
        "name": "s-secsvcAgBackendPool",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/backendAddressPools/s-secsvcAgBack
endPool",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "backendAddresses": [
            {
              "ipAddress": "10.206.21.201"
            }
          ],
          "requestRoutingRules": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/requestRoutingRules/s-secsvc
AgRequestRoutingRule"
            }
          ]
        }
      }
    ],
    "backendHttpSettingsCollection": [
      {
        "name": "s-secsvcAgBackendHttpSettings",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/backendHttpSettingsCollection/s-se
csvcAgBackendHttpSettings",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "port": 443,
          "protocol": "Https",
          "cookieBasedAffinity": "Disabled",
          "requestTimeout": 30,
          "probe": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/probes/s-secsvcAgProbe"
          },
          "authenticationCertificates": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/authenticationCertificates/s
-secsvcAgAuthenticationCert"
            }
          ],
          "requestRoutingRules": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/requestRoutingRules/s-secsvc
AgRequestRoutingRule"
            }
          ]
        }
      }
    ],
    "httpListeners": [
      {
        "name": "s-secsvcAgHttpListener",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/httpListeners/s-secsvcAgHttpListen
er",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "frontendIPConfiguration": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/frontendIPConfigurations/s-sec
svcAgFrontendIP"
          },
          "frontendPort": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/frontendPorts/s-secsvcAgFronte
ndPort"
          },
          "protocol": "Https",
          "sslCertificate": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/sslCertificates/s-secsvcAgSslC
ert"
          },
          "requireServerNameIndication": false,
          "requestRoutingRules": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/requestRoutingRules/s-secsvc
AgRequestRoutingRule"
            }
          ]
        }
      }
    ],
    "urlPathMaps": [],
    "requestRoutingRules": [
      {
        "name": "s-secsvcAgRequestRoutingRule",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/requestRoutingRules/s-secsvcAgRequ
estRoutingRule",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "ruleType": "Basic",
          "httpListener": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/httpListeners/s-secsvcAgHttpLi
stener"
          },
          "backendAddressPool": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/backendAddressPools/s-secsvcAg
BackendPool"
          },
          "backendHttpSettings": {
            "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/backendHttpSettingsCollection/
s-secsvcAgBackendHttpSettings"
          }
        }
      }
    ],
    "probes": [
      {
        "name": "s-secsvcAgProbe",
        "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/probes/s-secsvcAgProbe",
        "etag": "W/\"32093f62-12bc-467e-8604-b9eb0fd0ef47\"",
        "properties": {
          "provisioningState": "Succeeded",
          "protocol": "Https",
          "host": "127.0.0.1",
          "path": "/probe/",
          "interval": 15,
          "timeout": 120,
          "unhealthyThreshold": 2,
          "backendHttpSettings": [
            {
              "id": "/subscriptions/0967fd64-a807-47e1-bf05-0e55e24a68d5/resourceGroups/prism-s-secsvc/providers/Microsoft.Network/applicationGateways/s-secsvc-aag/backendHttpSettingsCollectio
n/s-secsvcAgBackendHttpSettings"
            }
          ]
        }
      }
    ]
  }
}

Get-AzureRmApplicationGateway : Missing type map configuration or unsupported mapping.
Mapping types:
Int32 -> UInt32
System.Int32 -> System.UInt32
Destination path:
PSApplicationGateway.Probes.Probes.Probes0[0].Interval.Interval
Source value:
15
At line:1 char:1
+ Get-AzureRmApplicationGateway -Name s-secsvc-aag -ResourceGroupName p ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzureRmApplicationGateway], AutoMapperMappingException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Network.GetAzureApplicationGatewayCommand
 
DEBUG: 10.31.06 - GetAzureApplicationGatewayCommand end processing.
DEBUG: 10.31.06 - GetAzureApplicationGatewayCommand end processing.

### Script/Steps for Reproduction

Execute Get-AzureRmApplicationGateway with or without parameters

**Comment by cormacpayne** [comment]
@DeepakRajendranMsft Hey Deepak, would you mind taking a look at this issue?

**Comment by inghak** [comment]
I got the AzureRmApplicationGateway functions to work. They work when I install Azure Powershell 3.3.0 using the msi (that installs them to C:\Program Files (x86)\Microsoft SDKs\Azure\PowerShell\ResourceManager\AzureResourceManager).

They do NOT work when I install AzureRM using Install-Module (PowerShellGet) that puts them in “C:\Program Files\WindowsPowerShell\Modules\...” Everything else other than ApplicationGateway stuff works fine in both scenarios.

I do not know why.

You find the AzureRM Powershell MSI here:
https://github.com/Azure/azure-powershell/releases

**Comment by alecor191** [comment]
Thanks @inghak for sharing the workaround of installing the MSI. I had the same issue. Using your workaround unblocked me as well. Thanks!

**Comment by mino-s2000** [comment]
+1
I had same issue.

Modules version (Installing from WebPI):
PS C:\> Get-Module -Name AzureRM.Network

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   3.4.0      AzureRM.Network                     {Add-AzureRmApplicationGatewayAuthenticationCertificate, Add-AzureRmApplicationGatewayBa...

Run "Get-AuzreRmApplicationGateway":
PS C:\>Get-AzureRmApplicationGateway -ResourceGroupName myrg -Name myappgw

Get-AzureRmApplicationGateway : Missing type map configuration or unsupported mapping.
Mapping types:
Int32 -> UInt32
System.Int32 -> System.UInt32
Destination path:
PSApplicationGateway.BackendHttpSettingsCollection.BackendHttpSettingsCollection.BackendHttpSettingsCollection0[0].RequestTimeout.RequestTimeout
Source value:
30
At line:1 char:1
+ Get-AzureRmApplicationGateway -ResourceGroupName myrg -Name myappgw
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzureRmApplicationGateway]、AutoMapperMappingException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Network.GetAzureApplicationGatewayCommand

**Comment by cormacpayne** [comment]
Closing this issue as it seems to be a duplicate of issue https://github.com/Azure/azure-powershell/issues/2873 , which we will use for tracking the status of the fix
